modules/proc_manage-repomirror.adoc

```diff
@@ -138,16 +138,14 @@ or no certificate.
 +
 * **HTTP Proxy**: Identify the proxy server needed to access the remote site, if one is required.
 +
-* **Tags**: This field is required. You can enter individual tags or indicate which
-tags to match using globbing or regular expressions. For example, you can match specific
-instances of an image (such as latest), enter a comma-separated list of tags, or use
-wildcards (such as mytag*) to match multiple images. For example,
-for a repository that features daily image builds named abc-2019-08-12, abc-2019-08-13,
-and so on, you could mirror all the August, 2019 images using abc-2019-08* as the tag.
+* **Tags**: This field is required. You may enter a comma-separated list of individual tags or
+tag patterns. (See section ??Tag Patterns?? for details.)
```

Comment:
@thomasmckay Oh, darn. Good question. Since we only had one location where docs were published, I just used full URLs. I guess we'll have to start identifying sections with "id=" and point to sections with "xref". I'll get back to you on this.